x-pack/test/security_solution_api_integration/test_suites/genai/knowledge_base/entries/trial_license_complete_tier/entries.ts

```diff
@@ -32,10 +34,11 @@ export default ({ getService }: FtrProviderContext) => {
   const es = getService('es');
   const ml = getService('ml') as ReturnType<typeof MachineLearningProvider>;
 
-  describe.skip('@ess Basic Security AI Assistant Knowledge Base Entries', () => {
+  describe('@ess Basic Security AI Assistant Knowledge Base Entries', () => {
     before(async () => {
       await installTinyElser(ml);
-      await setupKnowledgeBase(supertest, log);
+      setupKnowledgeBase(supertest, log);
+      await waitForKnowledgeBaseModelToBeDeployed(supertest, log);
```

Comment:
Does waiting for the knowledge base setup take up to 10 minutes? If so, I think we should find a way to not install all of the security docs for this test. Perhaps there is a way to use the test framework to mock the directory where the documents are installed from, and only install one document?

Reply:
Yeah, that will be a good option as well. I will explore this.

Reply:
By the way, below this there is an `afterEach` with `clearKnowledgeBase` that deletes all of the knowledge base documents, so it is extra silly to wait 10 minutes to install them. And `setupKnowledgeBase` only happens once